interop-testing/src/main/java/io/grpc/testing/integration/XdsTestClient.java

```diff
@@ -81,7 +82,8 @@
   private int qps = 1;
   private volatile RpcConfig rpcConfig;
   private int rpcTimeoutSec = 20;
-  private String server = "localhost:8080";
+  private boolean secureMode = false;
+  private String server = "localhost:8081";
```

Comment:
I think this should be reverted too
```suggestion
  private String server = "localhost:8080";
```